### ISSUE TITLE
where can I get the test dataset

### ISSUE THREAD

**Issue by expert668** [cersar/3D_detection#7]
I have tried to run detection.py, but I found that there is no testing labels and calib.txt in kitti datasets. Can you give me a link to download it

**Comment by cersar** [comment]
Hello,when i was training the pretrained model,I splited the origin training dataset to training and testing datasets by the ratio 5:1.I will upload the datasets file list later.

**Comment by expert668** [comment]
Thanks! One more important question is the calib file, why is your calib file a single one? when i down load the calibration_matrice file , I got plenty of .txt.

**Comment by cersar** [comment]
According to the calib files,i find the contents in these files almost the same.So,for simplicity,I use single calib file instead.

**Comment by expert668** [comment]
Thanks! your work is really great!

**Comment by cersar** [comment]
Hello,i just upload my train/test datesets file list in the dataset directory

**Comment by yanggui19891007** [comment]
> 我试过运行detect.py，但我发现kitti数据集中没有测试标签和calib.txt。你能给我一个下载它的链接吗？

**Comment by xueyingliu** [comment]
@yanggui19891007 
train.txt and test.txt are from [here](https://github.com/cersar/3D_detection/tree/master/dataset), and you can get calib.txt from [kitti webset](http://www.cvlibs.net/download.php?file=data_object_calib.zip).
